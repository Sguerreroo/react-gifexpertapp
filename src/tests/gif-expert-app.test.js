import { shallow } from 'enzyme';

import GifExpertApp from '../GifExpertApp';

describe('<GifExpertApp />', () => {

  let wrapper = shallow(<GifExpertApp />);

  beforeEach(() => {
    wrapper = shallow(<GifExpertApp />);
  });

  test('should render <GifExpertApp /> component successfully', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should render an h2 element', () => {
    const gifExpertAppTitle = wrapper.find('h2');

    expect(gifExpertAppTitle.length).toBe(1);
  });

  test('should render an h2 element with GifExpertApp text', () => {
    const gifExpertAppTitleText = wrapper.find('h2').text();

    expect(gifExpertAppTitleText).toBe('GifExpertApp');
  });

  test('should render an hr element', () => {
    const gifExpertAppTitle = wrapper.find('hr');

    expect(gifExpertAppTitle.length).toBe(1);
  });

});
