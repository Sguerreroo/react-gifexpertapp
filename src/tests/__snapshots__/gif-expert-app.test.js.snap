// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifExpertApp /> should render <GifExpertApp /> component successfully 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <hr />
</Fragment>
`;
